public/pages/CreateTrigger/components/Action/actions/Message.js

```diff
@@ -122,6 +123,49 @@ const Message = ({
       </EuiFormRow>
 
       <EuiSpacer size="s" />
+
+      <EuiFormRow
+          label={
+            <div>
+              <span>Action Throttling</span>
+              <EuiButtonEmpty
+                size="s"
+                onClick={() => { setFlyout({ type: 'messageFrequency' }); }}
+              >
+                Info
+              </EuiButtonEmpty>
+            </div>
+          }
+          style={{ maxWidth: '100%' }}
+        >
+          <EuiFlexGroup alignItems="center">
+            <EuiFlexItem grow={false} style={{ marginRight: '0px' }}>
+              <FormikSwitch
+                name={`actions.${index}.throttle_enabled`}
+                inputProps={{ label: 'Maximum Frequency' }}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem grow={false} style={{ marginLeft: '4px', marginRight: '4px' }}>
+              <FormikFieldNumber
+                name={`actions.${index}.throttle.value`}
+                fieldProps={{ validate: value => {
+                  if (value <= 0) return 'Needs to be a positive number';
+                } }}
+                inputProps={{
```

Comment:
Provide min value props to fieldValue

Reference [here](https://elastic.github.io/eui/#/forms/form-controls)

Reply:
Good suggestion. Will add it.